Calculator.py

```diff
@@ -1,7 +1,19 @@
 def gather_inputs():
     operation = input("Please type in operation you would like to complete: +, -, *, or /")
-    number_1 = int(input('Please enter the first number: '))
-    number_2 = int(input('Please enter the second number: '))
+    while (True):
```

Comment:
Exceptions for invalid user inputs would be very helpful and I like the format of your exception handling, as it is pretty consistent with the rest of the code.